### DISCUSSION TITLE
Attendance

### DISCUSSION THREAD

**Discussion by Sequner** [seominjoon/kaist-ai605#135] [Q&A] [answered]
Hello,

How can we check our attendance? It is my first semester at KAIST, so, I don't know

**Comment by seominjoon** [comment, marked as answer]
Hi, attendance is taken via quiz. Have you voted in the quiz polls?

**Reply by Sequner** [reply]
Sorry, what I meant is: where can I see the total number of lectures I have attended (in how many quiz polls I have voted)? I forgot to count :(

**Reply by miyoungko** [reply]
You can check your attendance in KLMS, at 'Attendance (~11.08)' tab.
